### ISSUE TITLE
Run command for wp-cli should still be there even if the cli is not installed

### ISSUE THREAD

**Issue by Chris2011** [junichi11/netbeans-wordpress-plugin#64]
#### Issue type

- [ ] Bug
- [x] Enhancement

#### Overview description
I expected to see the `Run command` action under the WordPress action sub-menu. It was my first try to find out, where I can find wp-cli or smth similar. So I searched under Right click -> WordPress. But I couldn't see it. So I read your readme and I figured it out, that I have to set the path inisde of the options. Now I see it, but in general, the action should be there even if the wp-cli is not installed. Maybe deactivated and if you hover on it, there should be a tooltip, please set it inside the option OR while clicking on Run command, it should open a dialog where it says: "Please set the wp-cli inside the options". With a link to it. This is a better UX. Sure, don't show things, that are not available, but I was confused to not find such wp-cli run command action.

So long story short, it will be better, to see the Run command action anyway and open a dialog, where it points the user to the options, when you click on it and wp-cli is not installed.

#### Steps to reproduce

1. Don't set wp-cli in the options
2. Right click on a WordPress project
3. Go to action WordPress

#### Actual results
No run command action is shown

#### Expected results
1. Show run command action
2. Click should open dialog with info text and link to the options

#### Your environment (NetBeans and plugin version, e.t.c.)
Windows 10, NetBeans 8.2, PHP 7.1.1, WordPress Plugin 0.9.2.

**Comment by junichi11** [comment]
Done. Give it a try. 0.9.3.1-dev
